fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindExpression.java

```diff
@@ -507,6 +508,31 @@ private LogicalJoin<Plan, Plan> bindJoin(MatchingContext<LogicalJoin<Plan, Plan>
         LogicalJoin<Plan, Plan> join = ctx.root;
         CascadesContext cascadesContext = ctx.cascadesContext;
 
+        Set<String> tableNames = new HashSet<>();
+        List<Plan> children = join.children();
+        for (Plan child : children) {
```

Comment:
this seems only check one level alias, can you support check conflict names between multiple level?